.github/workflows/stockfish_test.yml

```
@@ -119,8 +119,9 @@ jobs:
 
       - name: Extract the bench number from the commit history
         run: |
-          for ((n=0; n<100; n++)); do
-            benchref=$(git log HEAD~$n -1 | tac | grep -m 1 -o -x '[[:space:]]*\b[Bb]ench[ :]\+[1-9][0-9]\{5,7\}\b[[:space:]]*' | sed 's/[^0-9]//g') && break || true
+          for hash in $(git log --format=format:%H -100)
+          do
+            benchref=$(git log $hash -1 | tac | grep -m 1 -o -x '[[:space:]]*\b[Bb]ench[ :]\+[1-9][0-9]\{5,7\}\b[[:space:]]*' | sed 's/[^0-9]//g') && break || true
```

I think `git show` makes a bit more clear that we show just a single commit. (I missed the `-s` flag to suppress the diff output in my proposal on Discord.)

```suggestion
            benchref=$(git show -s $hash | tac | grep -m 1 -o -x '[[:space:]]*\b[Bb]ench[ :]\+[1-9][0-9]\{5,7\}\b[[:space:]]*' | sed 's/[^0-9]//g') && break || true
```